massa-protocol-worker/src/handlers/block_handler/retrieval.rs

```diff
@@ -933,6 +933,9 @@ impl RetrievalThread {
             .expect("could not compute next block retrieval timer tick");
 
         if self.asked_blocks.is_empty() && self.block_wishlist.is_empty() {
+            // Note: in mainnet and before ICO, no blocks are processed but the timer need to be updated
```

Comment:
not ICO but genesis, and it is not specific to mainnet

Reply:
need => needs

Reply:
done